pixi.toml

```diff
@@ -18,3 +18,4 @@ numpy = ">=1.26.4,<1.27"
 black = ">=24.4.2,<24.5"
 isort = ">=5.13.2,<5.14"
 jupyterlab-git = ">=0.50.0,<0.51"
+myst-nb = ">=1.1.0,<1.2"
```

Comment:
For the sorting in TOML files (and other formatting) use Taplo. You can enforce it with pre-commit:

```yaml
  - repo: https://github.com/redeboer/taplo-pre-commit
    rev: v0.9.1rc1
    hooks:
      - id: taplo
```

See https://github.com/tamasfe/taplo/pull/549 for why to use the fork for now.

I recommend adding [this](https://github.com/ComPWA/ampform-dpd/blob/main/.taplo.toml) [Taplo config](https://taplo.tamasfe.dev/configuration/file.html):

```toml
[formatting]
align_comments = false
align_entries = false
allowed_blank_lines = 1
array_auto_collapse = false
array_auto_expand = true
array_trailing_comma = true
column_width = 88
compact_inline_tables = true
indent_string = "    "
reorder_arrays = true
reorder_keys = true
```